### DISCUSSION TITLE
Bug in suggested values when using \cite{}

### DISCUSSION THREAD

**Discussion by Msadat97** [James-Yu/LaTeX-Workshop#3428] [Q&A]
Hello everyone. I have a question regarding the autocompletion of citation keys. When I use \cite{}, latex workshop shows me a bunch of snippets alongside the actual BibTeX keys. Do you know how I can solve this?

<img width="865" alt="Screenshot 2022-08-13 at 12 23 35" src="https://user-images.githubusercontent.com/71992433/184479597-21aedffb-dec4-4ba1-8824-e969a37883bb.png">

However, this does not happen if I manually type \cite{ instead of choosing it from the IntelliSense suggestions:

<img width="865" alt="Screenshot 2022-08-13 at 12 24 16" src="https://user-images.githubusercontent.com/71992433/184479618-fc6d6c91-5c85-4b24-a5ea-8049511d8e4c.png">

Thank you in advance for your help.

**Comment by jlelong** [comment]
You probably have `editor.suggest.showSnippets` set to `true`. Therefore, vscode mixes snippet suggestions and intellisense suggestions. Hence, the first screenshot. You can also play with `editor.snippetSuggestions`.

In the second screenshot, only LaTeX-Workshop intellisense is triggered. I do not think that we can control this difference of behaviour from the extension. 

See also https://stackoverflow.com/questions/38703868/hide-snippets-when-showing-the-intellisense-to-get-properties-in-vscode
